appium-uiautomator2-server/build.gradle

```diff
@@ -37,3 +42,21 @@ dependencies {
     androidTestCompile 'com.android.support.test.espresso:espresso-core:2.0'
 
 }
+
+task installAUT(type: Exec, dependsOn: 'installDebug') {
+    group = 'test'
+    description = 'Install app under test'
+    def adb = android.getAdbExe().toString()
+    def apk = file('../res/selendroid-test-app-debug.apk')
+    commandLine "$adb install -rg $apk".split(' ')
+}
+
+afterEvaluate {
+    // When launching individual tests from Android Studio, it seems that only the assemble tasks
+    // get called directly, not the install* versions
```

Comment:
if we're copying the wording, probably good to link to the original gist also so they don't complain later.